### PR TITLE
ISSITES-6006 App View Changes

### PR DESCRIPTION
**Pull request by liaoethan**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/DashboardPage/DashboardPage.tsx [hunk 1]
```diff
@@ -96,6 +104,17 @@ export function DashboardPage() {
...
+
+			const newAppNavigationItem = {
+                ...currentAppNavigationItem,
+                items: newAppList,
+            }
+
+			setDashboardNavigationItems([
+                ...dashboardNavigationItems.filter((navigationItem) => navigationItem.itemName !== 'apps'),
+                newAppNavigationItem,
+            ]);
```
Comment (jotchon) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/DashboardPage/DashboardPage.tsx:109]: For the Dashboard Navigation, the apps should show above members, account, and sales.

Comment (jotchon) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/DashboardPage/DashboardPage.tsx:109]: Switch the order in the array where "newAppNavigationItem" comes first

**Review by jotchon — COMMENTED** [review]
Allen mentioned that Account and Catalog do not have an association yet so we will wait for that feature. We would need to remove the catalog name from the Dashboard navigation.

Currently adding new app is not working.  Let's make sure that's working before moving forward.

Alphabetizing/Source Formatting all files touched.

Keeping Apps at the top of the Dashboard Navigation.

Let me know if you have questions!

**Comment by liaoethan** [comment]
Hi @jotchon !

Thanks for the the feedback. Just a reminder for future reviewers - add the lastUpdatedBy field to the Commerce Product (Control Panel -> Objects -> Fields). I added the changes for the other things that you mentioned. Let me know if I should send this over to Ryan, thanks!

**Comment by jotchon** [comment]
Hi @liaoethan !  This looks great!  Try to clean up commits if you can.  Otherwise, feel free to send over to Ryan.

**Pull request closed without merging**